contracts/oracle/src/contract/config.rs

```diff
@@ -60,7 +55,18 @@ mod tests {
             Usdc::TICKER.to_string(),
             60,
             Percent::from_percent(50),
-            TreeStore(tr((0, Usdc::TICKER.to_string())) / tr((1, Cro::TICKER.to_string()))),
+            // TreeStore(tr((0, Usdc::TICKER.to_string())) / tr((1, Cro::TICKER.to_string()))),
+            serde_json_wasm::from_str(&format!(
+                r#"{{
+                    "value":[0,"{usdc}"],
+                    "children":[
+                        {{"value":[1,"{cro}"]}}
+                    ]
+                }}"#,
+                usdc = Usdc::TICKER,
+                cro = Cro::TICKER,
+            ))
+            .unwrap(),
```

Comment:
this segment repeats a couple of times in this module .... pls address it

Reply:
Addressed.